### PR TITLE
Sort index releases by semver

### PR DESCRIPTION
**Pull request by fgimenez**
Aims to get the right versions for e2e `WIP` and `current` builds, sorting by date doesn't get the more recent versions of each kind.

### CODE REVIEW

File: pkg/framework/version_bundle_test.go~ [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package framework
+
+import "testing"
+
+func Test_SortBySemver(t *testing.T) {
+
+}
```
Comment (rossf7) [inline pkg/framework/version_bundle_test.go~]: Looks like this is a temp file.

File: pkg/framework/version_bundle.go [hunk 2]
```diff
@@ -6,6 +6,7 @@ import (
 	"log"
 	"sort"
 
+	"github.com/blang/semver"
```
Comment (rossf7) [inline pkg/framework/version_bundle.go]: I like the use of the semver library here.

**Review by rossf7 — APPROVED** [review]
LGTM apart from one temp file issue.

File: pkg/framework/version_bundle_test.go~ (hunk 1 above)
Comment (fgimenez) [reply pkg/framework/version_bundle_test.go~]: Very good catch thx! my global git ignore got lost, fixed

File: pkg/framework/version_bundle.go (hunk 2 above)
Comment (tuommaki) [reply pkg/framework/version_bundle.go]: I'm kind of late here, but is there something specific in that library compared to `github.com/coreos/go-semver/semver` used in `versionbundle`? FYI: There's also same implementation in `versionbundle` library: https://github.com/giantswarm/versionbundle/blob/master/releases_sort.go#L9 :)

Comment (fgimenez) [reply pkg/framework/version_bundle.go]: Absolutely, I can take care of that, already put the changes here to take advantage of that.

**Comment by fgimenez** [comment]
Updated to use the new IndexRelease sorter from versionbundle giantswarm/versionbundle#52 thx @tuommaki! Merging.

**Pull request merged**